drivers/syslog/Kconfig

```diff
@@ -111,6 +111,37 @@ config SYSLOG_TIMESTAMP_REALTIME
 		CLOCK_MONOTONIC, if enabled, will be used or the system timer
 		is not.
 
+config SYSLOG_TIMESTAMP_FORMATED
+	bool "Formatted syslog time"
+	default n
+	depends on SYSLOG_TIMESTAMP_REALTIME
```

Comment:
let's depends on SYSLOG_TIMESTAMP, so all time source can get the formated output. Other change looks good now, thanks.

Reply:
CONFIG_SYSLOG_TIMESTAMP_FORMATED... The correct spelling for formatted in this sense has two T's like CONFIG_SYSLOG_TIMESTAMP_FORMATTED

Reply:
@patacongo Fixed the typo.

@xiaoxiang781216 What would be the meaning of formatting the system clock with `strftime`? I think it is not meaningful.

Reply:
Ok, localtime_r and gmtime_r isn't good for system time, but it looks good to format the output from 61.000 to 00:01:01.000 for system time.